server/views/assessments/pages/risk-information/prison-information.njk

```diff
@@ -116,6 +116,31 @@
                 </div>
             </div>
 
+            <div class="govuk-column-full-width" id="acct" role="tabpanel" aria-labelledby="acctTab" hidden>
+                <table class="govuk-table">
```

Comment:
Does this show the same information as in Apply? If so, might be good to have this in a partial

Reply:
I've added a macro for the whole table which takes into account the differences in the 'Prison Case Notes' tab.